src/local/BLELocalDevice.h

```diff
@@ -43,7 +43,7 @@ class BLELocalDevice {
 
   void setAdvertisedServiceUuid(const char* advertisedServiceUuid);
   void setAdvertisedService(const BLEService& service);
-  void setManufacturerData(const uint8_t manufacturerData[], int manufacturerDataLength);
+  void setManufacturerData(const uint16_t companyId, const uint8_t manufacturerData[], int manufacturerDataLength);
```

Comment:
It would be good to maintain backwards compatibility here and support both the newer and older API.

Reply:
Thank you for pointing out the problem.
Please check new code.